src/applications/ezr/locales/en/content.json

```diff
@@ -54,14 +54,14 @@
   "military-service-warfare-agents-exposure-title": "Warfare agents (like nerve agents or chemical and biological weapons)",
   "military-service-other-exposure-title": "Other toxins or hazards not listed here",
   "military-service-other-exposure-dates-title": "Dates of exposure",
-  "military-service-other-exposure-dates-description": "Enter any date range when you were exposed to other toxins or hazards. You don\u2019t need to have exact dates.",
+  "military-service-other-exposure-dates-description": "Enter any date range when you think you may have been exposed to other toxins or hazards. You don\u2019t need to have exact dates.",
   "military-service-other-exposure-dates-start": "Exposure start date",
   "military-service-other-exposure-dates-end": "Exposure end date",
   "military-service-other-exposure-title-2": "Other toxic exposure",
-  "military-service-other-exposure-description": "You selected that you were exposed to other toxins or hazards.",
-  "military-service-other-exposure-description-2": "Enter any toxins or hazards you\u2019ve been exposed to",
+  "military-service-other-exposure-description": "You selected that you think you may have been exposed to other toxins or hazards.",
+  "military-service-other-exposure-description-2": "Enter any toxins or hazards you think you may have been exposed to",
   "military-service-other-exposure-description-2-hint": "If you\u2019re listing multiple toxins or hazards, list them without commas or any other special characters",
```

Comment:
Should this be removed since I don't see anywhere we are using it now?

Reply:
We use these in the `otherToxicExposureDetails.js` file.

Reply:
I thought that was just deleted in this PR?

Reply:
Nope it's overwritten. Still in use.

Reply:
Ohhhhh sorry for some reason it looked like you were talking about the `"military-service-other-exposure-description"` and `"military-service-other-exposure-description-2"`. You're right the hint text can be deleted. I will push that right up.